site/layouts/resources/single.html

```diff
@@ -3,12 +3,12 @@
   <div class="mw6 center ph3 pv4">
     <h1 class="f2 lh-title b mb3">{{.Title}}</h1>
     <div class="flex justify-between grey-3">
-      <p>{{ .Date.Format "Mon, Jan 2, 2006" }}</p>
+      <!-- <p>{{ .Date.Format "Mon, Jan 2, 2006" }}</p> -->
```

Comment:
nice catch! it's ok to delete this line.

Reply:
Hey I deleted that line for the date, a new review is required. Then if you're ok with everything I can merge it to master.